weave-js/src/core/model/graph/index.ts

```diff
@@ -2,6 +2,7 @@ export * from './construction';
 export * from './editing';
 export * from './norm';
 export * from './serialize';
+export * from './serialize2';
```

Comment:
nit: can we avoid `thing2` naming - maybe something that indicates the difference would be more helpful.

nit2: i think `serialize` and `serialize2` have a bunch of the same symbols. This import pattern (importing both) will end up with `serialize2`s + anything from `serialize` not overwrote by `serialize2`. Is this intended?

Reply:
Changed to `serializeAllValues`.

TypeScript complains when there are duplicate symbols exported, so there's no concept of "overwriting". Everything exported by `serialize2` (now `serializeAllValues`) had different names.